app/index.js

```diff
@@ -283,6 +283,9 @@ app.on('ready', () => {
     win.on('focus', () => {
       win.focusTime = process.uptime();
     });
+    // The focus event doesn't fire on win.show, but we need
+    // this to track the most recent window in focusedWindow
+    win.emit('focus');
```

Comment:
Please don't hate me!! 🙈 Only a small and maybe irrelevant thing about the comment.
`win.show` triggers focus correctly, according to the electron docs: but we trigger `win.show` too late for our purposes.